src/main/resources/com/hcl/appscan/jenkins/plugin/auth/ASoCCredentials/help-certificates_en.properties

```diff
@@ -0,0 +1,4 @@
+#NLS_MESSAGEFORMAT_ALL
+#NLS_ENCODING=UTF8
+
+description.disable.certificates = Allows the AppScan server to accept self-signed SSL server certificates.
```

Comment:
I suggest a description like: "Allows untrusted connections to the AppScan service" or something along those lines.

Reply:
Now, I am using "Allows untrusted connections to the AppScan service"